backend/src/apiserver/resource/model_converter.go

```diff
@@ -117,7 +166,6 @@ func (r *ResourceManager) ToModelRunDetail(run *apiv1beta1.Run, runId string, wo
 		}
 		runDetail.Parameters = params
 		runDetail.WorkflowSpecManifest = manifest
-		runDetail.WorkflowRuntimeManifest = workflow.ToStringForStore()
```

Comment:
Yes, since the info here comes from `workflow`, which hasn't been generated yet. It is patched in `updateModelRunWithNewScheduledWorkflow` function [here](https://github.com/kubeflow/pipelines/blob/a294a9d9c147a3761047b0bbdd14b9603aec489e/backend/src/apiserver/resource/model_converter.go#L238).